### ISSUE TITLE
Allows you to download and install a single homebrew multiple times

### ISSUE THREAD

**Issue by electr0sheep** [ArkSource/psphbb#13]
If there is a way that you could check to see if a particular homebrew is already installed (and maybe grey out the icon, or put a checkmark over it or something) that would be awesome. On the same token, if a homebrew is already installed, it would be awesome if we could choose to uninstall it with homebrew browser.

**Comment by ArkSource** [comment]
@electr0sheep 
You're right, Would be handy know what is already installed from the listing screen.
I'll mark this as an enhancement.

uninstalling PSP homebrew will be included with the [#4 HB Manager](https://github.com/ArkSource/psphbb/issues/4).
